tox.ini

```diff
@@ -21,6 +25,8 @@ commands =
     pep8: --application-import-names=constants,pubsub_utils
     nosetest: nosetests cmdline-pull
     nosetest: nosetests appengine-push/test_deploy.py
+    grpc: pip install -r requirements.txt --find-links https://gapi-pypi.appspot.com/admin/nurpc-dev
```

Comment:
Ok, I added README.md that describes that requirement. I also fixed the error in the test. It should pass this time.